src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -32,6 +33,22 @@ namespace ZoneWindowUtils
         }
         return std::wstring{ uniqueId };
     }
+
+    std::wstring GenerateUniqueIdAllMonitors(PCWSTR virtualDesktopId)

```

Comment:
GenerateUniqueIdAllMonitors -> GenerateUniqueIdAllMonitorsWorkArea

Reply:
Wouldn't that be misleading, as I'm generating it based on the monitor's entire area `.rcMonitor`, not its work area. Also, it wouldn't be in line with `GenerateUniqueId` right above it.

Reply:
GenerateUniqueIdAllMonitorsArea